<launch>
	
	<arg name="deviceId" default="1" />

	<node pkg="go1_republisher" name="camera_republisher" type="camera.py" output="screen">
		<param name="deviceId" value="$(arg deviceId)"/>	
	</node>
</launch>
